INFO: <2023-07-21 11:42>  Running stm_build_statistics([('self', <sai_engine.lib.web_scraper_bs4.webScraper object at 0x7f3b28be9f00>), ('stm_version', '8_0_1'), ('stm_build', '1223')])
INFO: <2023-07-21 11:42>  No Jenkins job mapping found for stm build 1223
INFO: <2023-07-21 11:42>  Running get_stm_build([('self', <sai_engine.lib.web_scraper_bs4.webScraper object at 0x7f3b28be9f00>), ('stm_version', '8_0_1'), ('script_name', 'alarms'), ('job_build_num', 2), ('log_file_path', '')])
INFO: <2023-07-21 11:42>  Running get_page_url([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('stm_version', '8_0_1'), ('script_name', 'alarms'), ('job_build_num', '2')])
INFO: <2023-07-21 11:42>  Validating script name alarms
INFO: <2023-07-21 11:42>  Running validate_script([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('stm_ver', '8_0_1'), ('script_name', 'alarms')])
INFO: <2023-07-21 11:42>  Page Url : /home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/2/log_alarms.html
INFO: <2023-07-21 11:42>  Running validate_log_file([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('full_path', '/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/2/log_alarms.html')])
INFO: <2023-07-21 11:42>  Running check_file_exists([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('full_path', '/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/2/log_alarms.html')])
ERROR: <2023-07-21 11:42>  Log file , /home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/2/log_alarms.html does not exist. Please recheck if file is downloaded
INFO: <2023-07-21 11:42>  Turns out the file [/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/2/log_alarms.html]does not exist...Downloading the files
INFO: <2023-07-21 11:42>  2048
INFO: <2023-07-21 11:42>  Running get_stm_build([('self', <sai_engine.lib.web_scraper_bs4.webScraper object at 0x7f3b28be9f00>), ('stm_version', '8_0_1'), ('script_name', 'alarms'), ('job_build_num', 4), ('log_file_path', '')])
INFO: <2023-07-21 11:42>  Running get_page_url([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('stm_version', '8_0_1'), ('script_name', 'alarms'), ('job_build_num', '4')])
INFO: <2023-07-21 11:42>  Validating script name alarms
INFO: <2023-07-21 11:42>  Running validate_script([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('stm_ver', '8_0_1'), ('script_name', 'alarms')])
INFO: <2023-07-21 11:42>  Page Url : /home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/4/log_alarms.html
INFO: <2023-07-21 11:42>  Running validate_log_file([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('full_path', '/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/4/log_alarms.html')])
INFO: <2023-07-21 11:42>  Running check_file_exists([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('full_path', '/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/4/log_alarms.html')])
ERROR: <2023-07-21 11:42>  Log file , /home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/4/log_alarms.html does not exist. Please recheck if file is downloaded
INFO: <2023-07-21 11:42>  Turns out the file [/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/4/log_alarms.html]does not exist...Downloading the files
INFO: <2023-07-21 11:42>  2048
INFO: <2023-07-21 11:42>  Running get_stm_build([('self', <sai_engine.lib.web_scraper_bs4.webScraper object at 0x7f3b28be9f00>), ('stm_version', '8_0_1'), ('script_name', 'alarms'), ('job_build_num', 5), ('log_file_path', '')])
INFO: <2023-07-21 11:42>  Running get_page_url([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('stm_version', '8_0_1'), ('script_name', 'alarms'), ('job_build_num', '5')])
INFO: <2023-07-21 11:42>  Validating script name alarms
INFO: <2023-07-21 11:42>  Running validate_script([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('stm_ver', '8_0_1'), ('script_name', 'alarms')])
INFO: <2023-07-21 11:42>  Page Url : /home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/5/log_alarms.html
INFO: <2023-07-21 11:42>  Running validate_log_file([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('full_path', '/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/5/log_alarms.html')])
INFO: <2023-07-21 11:42>  Running check_file_exists([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('full_path', '/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/5/log_alarms.html')])
ERROR: <2023-07-21 11:42>  Log file , /home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/5/log_alarms.html does not exist. Please recheck if file is downloaded
INFO: <2023-07-21 11:42>  Turns out the file [/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/5/log_alarms.html]does not exist...Downloading the files
INFO: <2023-07-21 11:42>  2048
INFO: <2023-07-21 11:42>  Running get_stm_build([('self', <sai_engine.lib.web_scraper_bs4.webScraper object at 0x7f3b28be9f00>), ('stm_version', '8_0_1'), ('script_name', 'alarms'), ('job_build_num', 6), ('log_file_path', '')])
INFO: <2023-07-21 11:42>  Running get_page_url([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('stm_version', '8_0_1'), ('script_name', 'alarms'), ('job_build_num', '6')])
INFO: <2023-07-21 11:42>  Validating script name alarms
INFO: <2023-07-21 11:42>  Running validate_script([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('stm_ver', '8_0_1'), ('script_name', 'alarms')])
INFO: <2023-07-21 11:42>  Page Url : /home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/6/log_alarms.html
INFO: <2023-07-21 11:42>  Running validate_log_file([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('full_path', '/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/6/log_alarms.html')])
INFO: <2023-07-21 11:42>  Running check_file_exists([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('full_path', '/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/6/log_alarms.html')])
ERROR: <2023-07-21 11:42>  Log file , /home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/6/log_alarms.html does not exist. Please recheck if file is downloaded
INFO: <2023-07-21 11:42>  Turns out the file [/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/6/log_alarms.html]does not exist...Downloading the files
INFO: <2023-07-21 11:42>  2048
INFO: <2023-07-21 11:42>  Running get_stm_build([('self', <sai_engine.lib.web_scraper_bs4.webScraper object at 0x7f3b28be9f00>), ('stm_version', '8_0_1'), ('script_name', 'alarms'), ('job_build_num', 7), ('log_file_path', '')])
INFO: <2023-07-21 11:42>  Running get_page_url([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('stm_version', '8_0_1'), ('script_name', 'alarms'), ('job_build_num', '7')])
INFO: <2023-07-21 11:42>  Validating script name alarms
INFO: <2023-07-21 11:42>  Running validate_script([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('stm_ver', '8_0_1'), ('script_name', 'alarms')])
INFO: <2023-07-21 11:42>  Page Url : /home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/7/log_alarms.html
INFO: <2023-07-21 11:42>  Running validate_log_file([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('full_path', '/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/7/log_alarms.html')])
INFO: <2023-07-21 11:42>  Running check_file_exists([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('full_path', '/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/7/log_alarms.html')])
ERROR: <2023-07-21 11:42>  Log file , /home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/7/log_alarms.html does not exist. Please recheck if file is downloaded
INFO: <2023-07-21 11:42>  Turns out the file [/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/7/log_alarms.html]does not exist...Downloading the files
INFO: <2023-07-21 11:42>  2048
INFO: <2023-07-21 11:42>  Running get_stm_build([('self', <sai_engine.lib.web_scraper_bs4.webScraper object at 0x7f3b28be9f00>), ('stm_version', '8_0_1'), ('script_name', 'alarms'), ('job_build_num', 8), ('log_file_path', '')])
INFO: <2023-07-21 11:42>  Running get_page_url([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('stm_version', '8_0_1'), ('script_name', 'alarms'), ('job_build_num', '8')])
INFO: <2023-07-21 11:42>  Validating script name alarms
INFO: <2023-07-21 11:42>  Running validate_script([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('stm_ver', '8_0_1'), ('script_name', 'alarms')])
INFO: <2023-07-21 11:42>  Page Url : /home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/8/log_alarms.html
INFO: <2023-07-21 11:42>  Running validate_log_file([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('full_path', '/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/8/log_alarms.html')])
INFO: <2023-07-21 11:42>  Running check_file_exists([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('full_path', '/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/8/log_alarms.html')])
ERROR: <2023-07-21 11:42>  Log file , /home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/8/log_alarms.html does not exist. Please recheck if file is downloaded
INFO: <2023-07-21 11:42>  Turns out the file [/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/8/log_alarms.html]does not exist...Downloading the files
INFO: <2023-07-21 11:42>  2048
INFO: <2023-07-21 11:42>  Running get_stm_build([('self', <sai_engine.lib.web_scraper_bs4.webScraper object at 0x7f3b28be9f00>), ('stm_version', '8_0_1'), ('script_name', 'alarms'), ('job_build_num', 9), ('log_file_path', '')])
INFO: <2023-07-21 11:42>  Running get_page_url([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('stm_version', '8_0_1'), ('script_name', 'alarms'), ('job_build_num', '9')])
INFO: <2023-07-21 11:42>  Validating script name alarms
INFO: <2023-07-21 11:42>  Running validate_script([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('stm_ver', '8_0_1'), ('script_name', 'alarms')])
INFO: <2023-07-21 11:42>  Page Url : /home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/9/log_alarms.html
INFO: <2023-07-21 11:42>  Running validate_log_file([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('full_path', '/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/9/log_alarms.html')])
INFO: <2023-07-21 11:42>  Running check_file_exists([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('full_path', '/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/9/log_alarms.html')])
ERROR: <2023-07-21 11:42>  Log file , /home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/9/log_alarms.html does not exist. Please recheck if file is downloaded
INFO: <2023-07-21 11:42>  Turns out the file [/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/9/log_alarms.html]does not exist...Downloading the files
INFO: <2023-07-21 11:42>  2048
INFO: <2023-07-21 11:42>  Running get_stm_build([('self', <sai_engine.lib.web_scraper_bs4.webScraper object at 0x7f3b28be9f00>), ('stm_version', '8_0_1'), ('script_name', 'alarms'), ('job_build_num', 10), ('log_file_path', '')])
INFO: <2023-07-21 11:42>  Running get_page_url([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('stm_version', '8_0_1'), ('script_name', 'alarms'), ('job_build_num', '10')])
INFO: <2023-07-21 11:42>  Validating script name alarms
INFO: <2023-07-21 11:42>  Running validate_script([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('stm_ver', '8_0_1'), ('script_name', 'alarms')])
INFO: <2023-07-21 11:42>  Page Url : /home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/10/log_alarms.html
INFO: <2023-07-21 11:42>  Running validate_log_file([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('full_path', '/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/10/log_alarms.html')])
INFO: <2023-07-21 11:42>  Running check_file_exists([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('full_path', '/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/10/log_alarms.html')])
ERROR: <2023-07-21 11:42>  Log file , /home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/10/log_alarms.html does not exist. Please recheck if file is downloaded
INFO: <2023-07-21 11:42>  Turns out the file [/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/10/log_alarms.html]does not exist...Downloading the files
INFO: <2023-07-21 11:42>  2048
INFO: <2023-07-21 11:42>  Running get_stm_build([('self', <sai_engine.lib.web_scraper_bs4.webScraper object at 0x7f3b28be9f00>), ('stm_version', '8_0_1'), ('script_name', 'alarms'), ('job_build_num', 11), ('log_file_path', '')])
INFO: <2023-07-21 11:42>  Running get_page_url([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('stm_version', '8_0_1'), ('script_name', 'alarms'), ('job_build_num', '11')])
INFO: <2023-07-21 11:42>  Validating script name alarms
INFO: <2023-07-21 11:42>  Running validate_script([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('stm_ver', '8_0_1'), ('script_name', 'alarms')])
INFO: <2023-07-21 11:42>  Page Url : /home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/11/log_alarms.html
INFO: <2023-07-21 11:42>  Running validate_log_file([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('full_path', '/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/11/log_alarms.html')])
INFO: <2023-07-21 11:42>  Running check_file_exists([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('full_path', '/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/11/log_alarms.html')])
ERROR: <2023-07-21 11:42>  Log file , /home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/11/log_alarms.html does not exist. Please recheck if file is downloaded
INFO: <2023-07-21 11:42>  Turns out the file [/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/11/log_alarms.html]does not exist...Downloading the files
INFO: <2023-07-21 11:42>  2048
INFO: <2023-07-21 11:42>  Running get_stm_build([('self', <sai_engine.lib.web_scraper_bs4.webScraper object at 0x7f3b28be9f00>), ('stm_version', '8_0_1'), ('script_name', 'alarms'), ('job_build_num', 12), ('log_file_path', '')])
INFO: <2023-07-21 11:42>  Running get_page_url([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('stm_version', '8_0_1'), ('script_name', 'alarms'), ('job_build_num', '12')])
INFO: <2023-07-21 11:42>  Validating script name alarms
INFO: <2023-07-21 11:42>  Running validate_script([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('stm_ver', '8_0_1'), ('script_name', 'alarms')])
INFO: <2023-07-21 11:42>  Page Url : /home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/12/log_alarms.html
INFO: <2023-07-21 11:42>  Running validate_log_file([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('full_path', '/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/12/log_alarms.html')])
INFO: <2023-07-21 11:42>  Running check_file_exists([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('full_path', '/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/12/log_alarms.html')])
ERROR: <2023-07-21 11:42>  Log file , /home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/12/log_alarms.html does not exist. Please recheck if file is downloaded
INFO: <2023-07-21 11:42>  Turns out the file [/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/12/log_alarms.html]does not exist...Downloading the files
INFO: <2023-07-21 11:42>  2048
INFO: <2023-07-21 11:42>  Running get_stm_build([('self', <sai_engine.lib.web_scraper_bs4.webScraper object at 0x7f3b28be9f00>), ('stm_version', '8_0_1'), ('script_name', 'alarms'), ('job_build_num', 13), ('log_file_path', '')])
INFO: <2023-07-21 11:42>  Running get_page_url([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('stm_version', '8_0_1'), ('script_name', 'alarms'), ('job_build_num', '13')])
INFO: <2023-07-21 11:42>  Validating script name alarms
INFO: <2023-07-21 11:42>  Running validate_script([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('stm_ver', '8_0_1'), ('script_name', 'alarms')])
INFO: <2023-07-21 11:42>  Page Url : /home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/13/log_alarms.html
INFO: <2023-07-21 11:42>  Running validate_log_file([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('full_path', '/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/13/log_alarms.html')])
INFO: <2023-07-21 11:42>  Running check_file_exists([('self', <sai_engine.lib.scraper_utilities.webScraper_utilities object at 0x7f3b28be9ab0>), ('full_path', '/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/13/log_alarms.html')])
ERROR: <2023-07-21 11:42>  Log file , /home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/13/log_alarms.html does not exist. Please recheck if file is downloaded
INFO: <2023-07-21 11:42>  Turns out the file [/home/amith/sailytics/production_server/sailytics-django/sai_engine/jenkins_logs/8_0_1/13/log_alarms.html]does not exist...Downloading the files
